src/net/rtsp/server.cpp

```diff
@@ -104,44 +109,122 @@ void Server::receive_request(ClientPos client_pos) {
         return;
       }
       client.m_sent_bytes = 0;
-      send_response(client_pos);
+      proccess_request(client_pos, request);
```

Comment:
`process_request()` can just return `Response`, so `m_response` field can be removed from `Client`.